mergetools/kdiff3

```diff
@@ -25,3 +25,7 @@ merge_cmd () {
 exit_code_trustable () {
 	true
 }
+
```

Comment:
Commenting on this line, as I cannot comment directly on the commit message:

- The message claims that kdiff3 is available for Windows, but it does not provide any download link. Maybe you want to add that?
- When explaining that the executable name has to be translated, maybe mention which code you imitated to fix this.
- There is a typo in "_thse_ operating systems"
- I seem to remember that `type -p kdiff3 >/dev/null 2>&1` was preferred over `which kdiff3`, can you compare to other merge tools?